backend/src/test/java/org/eclipse/tractusx/semantics/registry/AbstractAssetAdministrationShellApi.java

```diff
@@ -45,12 +45,12 @@
 @EnableConfigurationProperties( RegistryProperties.class)
 public abstract class AbstractAssetAdministrationShellApi {
 
-    protected static final String SHELL_BASE_PATH = "/api/v3.0/shell-descriptors";
-    protected static final String SINGLE_SHELL_BASE_PATH = "/api/v3.0/shell-descriptors/{aasIdentifier}";
-    protected static final String LOOKUP_SHELL_BASE_PATH = "/api/v3.0/lookup/shells";
-    protected static final String SINGLE_LOOKUP_SHELL_BASE_PATH = "/api/v3.0/lookup/shells/{aasIdentifier}";
-    protected static final String SUB_MODEL_BASE_PATH = "/api/v3.0/shell-descriptors/{aasIdentifier}/submodel-descriptors";
-    protected static final String SINGLE_SUB_MODEL_BASE_PATH = "/api/v3.0/shell-descriptors/{aasIdentifier}/submodel-descriptors/{submodelIdentifier}";
+    protected static final String SHELL_BASE_PATH = "/dec-registry/api/v3.0/shell-descriptors";
```

Comment:
Why we have the dec-registry here? 
It changes the API Path or not?

Reply:
Hi @tunacicek , This is fix for failing tests after adding the context root in the open api yaml file.

Reply:
@shijinrajbosch  yes, but i think, also the api path will be change. dec-registry could not be a part of the api.

Reply:
> @shijinrajbosch yes, but i think, also the api path will be change. dec-registry could not be a part of the api.

Thanks @tunacicek for the information. I made the context root placeholder generic.